webroot/src/models/Licensee/Licensee.model.ts

```diff
@@ -262,14 +276,22 @@ export class LicenseeSerializer {
             firstName: json.givenName,
             middleName: json.middleName,
             lastName: json.familyName,
-            homeJurisdiction: new State({ abbrev: json.homeJurisdictionSelection?.jurisdiction }),
-            address: AddressSerializer.fromServer({
+            // If the user has registered, json.homeJurisdictionSelection will be populated with the user's selected home state
+            // licenselicenseJurisdiction is the server's best guess at their home state. Once #467 is merged we can simply use
+            // json.licenselicenseJurisdiction as this will be updated to match the users' choice once that happens, therefor always
+            // being the best choice for this field. Also json.homeJurisdictionSelection is not available in get all responses
+            homeState: json.homeJurisdictionSelection
+                ? new State({ abbrev: json.homeJurisdictionSelection.jurisdiction })
+                : new State({ abbrev: json.licenseJurisdiction }),
+            // This value is updated to equal bestHomeStateLicenseMailingAddress() whenever a License record is added or updated for the user
+            // in the edge case where the user's best home state license expires this can get out of sync with that calulated value
```

Comment:
What's the implication of this being out of sync? And why would it get out of sync?

Also, nit, but the lack of punctuation threw me off a bit :) 
```suggestion
            // This value is updated to equal bestHomeStateLicenseMailingAddress() whenever a License record is added or updated for the user.
            // In the edge case where the user's best home state license expires, this can get out of sync with that calculated value.
```

Reply:
The implication is just that this the users address data can be stale by some definitions. This data is only used when should in the LicenseeDetail page

Reply:
Got it- thank you!